### ISSUE TITLE
Vaccine dispensing: "ok & next" button

### ISSUE THREAD

**Issue by mark-prins** [openmsupply/mobile#3681]
## Is your feature request related to a problem? Please describe.

Add an 'ok and next' button to the end of the process

## Describe the solution you'd like

This is to allow a user to finalise the current prescription and immediately create another

## Implementation

The current design has `Cancel` and `Confirm` buttons.
Add a third button which is called `Confirm & Next` to be right most, on the assumption that this is the most useful button. Cancel on the left, Confirm in the middle.

## Describe alternatives you've considered

There is an additional click required without using this button. Not doing anything is only a minor difference
It is a slightly more confusing UI - is the saving of a button press worth it?
Should the 'Confirm' be 'Ok' on both buttons?

## Additional context

Wording and button position up for debate

**Comment by joshxg** [comment]
I agree that the `OK & Next` should be a nice to have and be in a future version given it's saving a single button press. Unfortunately it seems as though it is now a requirement, from an email:

`The app will remember the vaccinator & site from the last patient and apply it to the next one by default, so you don’t have to re-enter (assuming that most often the next patient will have the same site/vaccinator as the last patient?)` 

.. 🙄

**Comment by joshxg** [comment]
blocked by https://github.com/openmsupply/mobile/issues/3682
